### PR TITLE
Support for new token (oidc-op)

### PR DESCRIPTION
**Pull request by jaxoncreed**
The support for new token effort contains the following PRs
 -  NSS (https://github.com/solid/node-solid-server/pull/1427)
 - oidc-auth-manager (https://github.com/solid/oidc-auth-manager/pull/48)
 - oidc-op (https://github.com/solid/oidc-op/pull/22)
 - oidc-rs (https://github.com/solid/oidc-rs/pull/5)

### CODE REVIEW

File: src/DpopAccessToken.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        payload = jwt.payload
+
+        return jwt.encode()
+      })
+      // set the response properties
+      .then(compact => {
+        response['access_token'] = compact
+        response['token_type'] = 'Bearer'
+        response['expires_in'] = max
+      })
```
Comment (NSeydoux) [inline src/DpopAccessToken.js:131]: This is not asynchronous code, what does it bring to break it up into a promise chain ?

File: src/DpopIDToken.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    let jwt = new DpopIDToken({ header, payload, key })
+
+    return jwt
+  }
+
+  /**
+   * issueForRequest
+   */
+  static issueForRequest (request, response) {
+    // TODO: Implement id_vc
```
Comment (NSeydoux) [inline src/DpopIDToken.js:73]: Would it be possible/beneficial to add some context to this TODO ?

File: src/Provider.js [hunk 3]
```diff
@@ -34,6 +35,8 @@ const DEFAULT_GRANT_TYPES_SUPPORTED = [
 ]
 const DEFAULT_SUBJECT_TYPES_SUPPORTED = ['public']
 
+const DEFAULT_TOKEN_TYPES_SUPPORTED = ['legacyPop', 'dpop']
```
Comment (NSeydoux) [inline src/Provider.js:38]: Should the constants defined in `src/handlers/AuthenticationRequest.js` be used here ?

**Comment by jaxoncreed** [comment]
@NSeydoux I've updated this PR to handle the bug

**Pull request merged**